### ISSUE TITLE
Parse error when using browser version of N3.js

### ISSUE THREAD

**Issue by warpr** [RubenVerborgh/N3.js#35]
Hello Ruben,

I ran into an issue today where the browserified version of N3.js fails to parse a file, but the "normal" N3.js has no trouble with that file.

The browserified version of N3 fails to parse this in Firefox, Chrome and under node.js, so it doesn't seem like a browser specific issue.

I'm testing with a reasonably large file (64577 triples).  Could this be a bug or am I running into a limitation of the browserify environment?

The stack trace I get is:
```
Syntax error: unexpected ""To" on line 9873.
    at Object.N3Lexer._syntaxError (/home/warp/www/n3/node_modules/n3/browser/n3-browser.js:415:12)
    at reportSyntaxError (/home/warp/www/n3/node_modules/n3/browser/n3-browser.js:382:54)
    at Object.N3Lexer._tokenizeToEnd (/home/warp/www/n3/node_modules/n3/browser/n3-browser.js:230:20)
    at Object._onImmediate (/home/warp/www/n3/node_modules/n3/browser/n3-browser.js:430:38)
    at processImmediate [as _immediateCallback] (timers.js:336:15)
```

I have a test case at https://gist.github.com/warpr/c4da9a48b9cb8f11a550 , "test.html" shows the bug in a browser, "test.js" shows it under node.js.

If I truncate the licensedb.2014-01-19.ttl file slightly before the syntax error there is no trouble and I can query the N3 Store as I'd expect.

**Comment by RubenVerborgh** [comment]
Very interesting. Will look at this in detail.

**Comment by RubenVerborgh** [comment]
The shortest failing case I've found is

    <a> <b> "\n".

Investigating now why it fails in the browser version.

**Comment by RubenVerborgh** [comment]
Found it: browserify doesn't isolate N3Lexer and N3Writer, so that `escapeReplacements` is overwritten.
Now let's see how to fix that.

**Comment by RubenVerborgh** [comment]
Fixed in 8df7eee3677ca53ac7ec6a9a6edc3a3b0b29ca10. Can you verify whether this works for you?
I've tested the gist and it worked fine.

Note that only the N3-generated browser version was affected; the browserify version worked alright.

**Comment by warpr** [comment]
Thanks Ruben!, this indeed fixes the problem.

(I didn't realize n3-browser.js and n3-browserify.js were different things, sorry for the confusion).

**Comment by RubenVerborgh** [comment]
Great, thanks for reporting this. Now available in v0.4.1.

**Comment by RubenVerborgh** [comment]
PS n3-browser is a simple build which more or less concatenates the source files. This is the version I recommend for standalone usage. n3-browserify is the Browserify-generated version, which has a bit more boilerplate, but integrates nicely with other Browserified libraries (for instance, as a dependency).
